docs/examples/use_cases/mxnet/mxnet-resnet50.ipynb

```diff
@@ -64,13 +64,19 @@
     "                                          pad_last_batch=is_training, name=\"Reader\")\n",
     "        dali_device = 'cpu' if dali_cpu else 'gpu'\n",
     "        decoder_device = 'cpu' if dali_cpu else 'mixed'\n",
+    "        # ask nvJPEG to preallocate memory for the biggest sample in ImageNet for CPU and GPU to avoid reallocations in runtime\n",
     "        device_memory_padding = 211025920 if decoder_device == 'mixed' else 0\n",
     "        host_memory_padding = 140544512 if decoder_device == 'mixed' else 0\n",
+    "        # as HW NVJPEG to allocate memory ahead for the biggest image in the data set to avoid reallocations in runtime\n",
```

Comment:
```suggestion
    "        # ask HW NVJPEG to allocate memory ahead for the biggest image in the data set to avoid reallocations in runtime\n",
```
My apologies for being nitpicky here, but there's a typo